### ISSUE TITLE
Can't create column with type 'text'.

### ISSUE THREAD

**Issue by JoshuaFlood** [dbeaver/dbeaver#6966]
#### System information: 
- Ubuntu 18.04
- DBeaver 6.2.2
- No additional extensions

#### Connection specification:
- PostgreSQL 10.10
- PostgreSQL JDBC Driver
- Connect via SSH proxy

#### Describe the problem you're observing:

When I attempt to create or import a table column with the data type `text`, I receive the following error response:

    SQL Error [42601]: ERROR: type modifier is not allowed for type "text"

#### Steps to reproduce, if exist:

 - Connect to an empty PostgreSQL database via SSH with DBeaver.
 - Create a table with a column with the `text` data type.
 - Save changes.
 - Dialogue window appears with error response.

#### Include any warning/errors/backtraces from the logs

<hr>

**Plug-in:** `org.jkiss.dbeaver.model`

**Error message:** `SQL Error [42601]: ERROR: type modifier is not allowed for type "text"
  Position: 45`

**Stack trace:** None available

**Session data:**

    eclipse.buildId=unknown
    java.version=11.0.3
    java.vendor=Oracle Corporation
    BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
    Command-line arguments:  -os win32 -ws win32 -arch x86_64
    
    org.jkiss.dbeaver.model
    Error
    Fri Oct 11 11:40:13 BST 2019
    SQL Error [42601]: ERROR: type modifier is not allowed for type "text"
      Position: 45
<hr>

**Comment by JoshuaFlood** [comment]
This bug existed in previous versions, see #1832 .

**Comment by JoshuaFlood** [comment]
Attempted the following workaround:

 - Open the 'columns' page of the table.
 - Right-click on the column in question and then select 'Edit column...'.
 - Using the dropdown menu that appears in the following dialogue, select 'varchar'.
 - Save changes with the floppy disk icon in the bottom-right corner of the DBeaver UI.

I receive the following, presumably related, error response:

    SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CHARACTER SET latin1 COLLATE latin1_swedish_ci NULL' at line 1

(Note that I have attempted various collations including utf8_general_ci and latin1_general_ci, just in case there was some superficial bug in the UI causing problems)

Session data:

    eclipse.buildId=unknown
    java.version=11.0.3
    java.vendor=Oracle Corporation
    BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
    Command-line arguments:  -os win32 -ws win32 -arch x86_64
    
    org.jkiss.dbeaver.model
    Error
    Fri Oct 11 12:42:07 BST 2019
    SQL Error [1064] [42000]: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'CHARACTER SET latin1 COLLATE latin1_swedish_ci NULL' at line 1

**Comment by serge-rider** [comment]
I can't reproduce this as I don't understand the use-case. 

- Is this about PostgreSQL or about MySQL?
- Do you perform simple table edit by adding new columns or you do some data transfer, import/export, backup/restore?

**Comment by JoshuaFlood** [comment]
@serge-rider

It's about PostgreSQL.

Originally I was trying to import/export data between databases, where the tables within the target database would be newly created using the DBeaver export wizard.

However, the explanation above is based on a simple edit on a new table within the default public schema which I performed in an attempt to debug this issue and establish whether it was me or the software that was making mistakes. I had not made any changes to the schema and all changes were made using the GUI, so I feel like the software is responsible for this issue.

One possibility I have considered, but have not tested, is that some residual files from previous DBeaver installations are causing a conflict, as this was an issue in previous versions.

I upgraded from the 6.2.1, but when I previously installed 6.2.1 it was a fresh install. I removed whichever previous version using Revo Uninstaller, to ensure that residual files and registry entries were all removed properly.. so unless there is something being left in my app data that your native uninstaller cannot handle, I don't feel that this is a likely cause.

I have not yet attempted to re-install the software in case you want me to find any logs or perform any troubleshooting, but I will attempt it once you've got no more questions.
